apps/remix-ide/src/assets/css/themes/remix-black_undtds.css

```diff
@@ -1,6 +1,6 @@
 @import url('https://fonts.googleapis.com/css?family=Nunito+Sans:400,600&display=swap');
 :root {
-  --blue: #28282D;
+  --blue: #568acf;
```

Comment:
could you please use #90c3f6; instead? the color you picked is too close to primary which is used for other keywords

Reply:
Thank you for pointing this out @LianaHus. I have made the change now.